depthai_ros_driver/include/depthai_ros_driver/dai_nodes/stereo.hpp

```diff
@@ -53,8 +56,8 @@ class Stereo : public BaseNode {
                     rclcpp::Node* node,
                     std::shared_ptr<dai::Pipeline> pipeline,
                     std::shared_ptr<dai::Device> device,
-                    StereoSensorInfo leftInfo = StereoSensorInfo{"left", dai::CameraBoardSocket::LEFT},
-                    StereoSensorInfo rightInfo = StereoSensorInfo{"right", dai::CameraBoardSocket::RIGHT});
+                    StereoSensorInfo leftInfo = StereoSensorInfo{"left", dai::CameraBoardSocket::CAM_B},
+                    StereoSensorInfo rightInfo = StereoSensorInfo{"right", dai::CameraBoardSocket::CAM_C});
```

Comment:
can we get this from the device getConnectedCameraProperties ?

Reply:
applies for everywhere CAM_B and CAM_C is used for for left and right.

Reply:
No. IIRC we have some new devices which has different ports.

Reply:
Updated with getConnectedCameraProperties()